### PR TITLE
Add support on consistent API for all the library components for class name and disabled props

### PR DESCRIPTION
**Pull request by hadasfa**
add support on consistent API for all the library components for class name and disabled props

#### Basic
- [ ] Used plop (`npm run plop`) to create a new component.
- [ V] PR has description.
- [ ] New component is functional and uses Hooks. 
- [ ] Component defines [`PropTypes`](https://reactjs.org/docs/typechecking-with-proptypes.html).
#### Style
- [ ] Styles are added to `NewComponent.modules.scss` file inside of the `NewComponent` folder.
- [ ] Component uses CSS Modules.
- [ ] Design is compatible with [Monday Design System](https://design.monday.com/).
#### Storybook
- [ ] Stories were added to `/src/NewComponent/__stories__/NewComponent.stories.js` file.
- [ ] Stories include all flows of using the component.
#### Tests
- [ ] Tests are compliant with [TESTING_README.md](TESTING_README.md) instructions.

### CODE REVIEW

File: src/components/Avatar/__stories__/avatar.stories.mdx [hunk 1]
```diff
@@ -74,9 +74,9 @@ Avatars appear in 3 sizes: Small, Medium, and Large.
 Use when a user is inactive in the system.
 <Canvas>
   <Story name="Disable">
-    <Avatar size={Avatar.sizes.SMALL} src={person1} type={Avatar.types.IMG} isDisabled />
-    <Avatar size={Avatar.sizes.MEDIUM} src={person1} type={Avatar.types.IMG} isDisabled />
-    <Avatar size={Avatar.sizes.LARGE} src={person1} type={Avatar.types.IMG} isDisabled />
+    <Avatar size={Avatar.sizes.SMALL} src={person1} type={Avatar.types.IMG} disabled />
```
Comment (sahariko) [inline src/components/Avatar/__stories__/avatar.stories.mdx:77]: What about `isSquare` stories?

File: src/components/BreadcrumbsBar/BreadcrumbItem/__stories__/breadcrumbItem.stories.mdx [hunk 2]
```diff
@@ -68,7 +68,7 @@ export const breadcrumbItemTemplate = createComponentTemplate(BreadcrumbItem)
       <div className="monday-storybook-breadcrumb-item_row-wrapper">
         <span>Disabled</span>
         <BreadcrumbsBar>
-          <BreadcrumbItem text="Workspace" icon={Workspace} isDisabled />
+          <BreadcrumbItem text="Workspace" icon={Workspace} disabled={} />
```
Comment (sahariko) [inline src/components/BreadcrumbsBar/BreadcrumbItem/__stories__/breadcrumbItem.stories.mdx:71]:
```suggestion
          <BreadcrumbItem text="Workspace" icon={Workspace} disabled />
```

File: src/components/Counter/Counter.jsx [hunk 3]
```diff
@@ -118,7 +124,7 @@ Counter.kinds = COUNTER_TYPES;
 Counter.propTypes = {
   /** id to pass to the element */
   id: PropTypes.string,
-  wrapperClassName: PropTypes.string,
+  className: PropTypes.string,
```
Comment (sahariko) [inline src/components/Counter/Counter.jsx:127]: Don't forget to update this component's stories.

File: src/components/Label/Label.jsx [hunk 4]
```diff
@@ -25,15 +36,15 @@ const Label = ({ wrapperClassName, kind, color, text = "", isAnimationDisabled,
 };
 
 Label.propTypes = {
-  wrapperClassName: PropTypes.string,
+  className: PropTypes.string,
```
Comment (sahariko) [inline src/components/Label/Label.jsx:39]: Don't forget to update this component's stories.

File: src/components/Loader/Loader.jsx [hunk 5]
```diff
@@ -1,23 +1,34 @@
...
+const Loader = forwardRef(
+  (
+    {
+      // Backward compatibility for props naming
+      svgClassName,
+      className
+    },
+    ref
+  ) => {
+    const overrideClassName = backwardCompatibilityForProperties([className, svgClassName], "");
```
Comment (hadasfa) [inline src/components/Loader/Loader.jsx:15]: undefined create here bad class name in {`circle-loader-spinner ${overrideClassName}` when empty

**Review by sahariko — APPROVED** [review]

**Pull request merged**